admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -501,10 +532,10 @@ Execution example:
    Please note, the execution of this command may take a long time for big
    AIP and Transfer Backlog storage locations, especially if the packages are
    stored compressed or encrypted, or you are using a third party service. If
-   that is the case, you may want to consider using the
-   :ref:`Update the Elasticsearch mappings <update-mappings>` approach instead.
+   that is the case, you may want to :ref:`reindexing
```

Comment:
I'd suggest changing "reindexing" to "reindex" here

Reply:
Thank you for your comments!